docs/react-18-and-react-native.md

```diff
@@ -0,0 +1,119 @@
+---
+id: react-18-and-react-native
+title: React 18 & React Native
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+This page describes how to use React 18 with React Native using the New React Native Architecture.
+
+> **tl;dr:** The first version of React Native compatible with React 18 is **0.69.0**. In order to use the new features in React 18 including automatic batching, `startTransition`, and `useDeferredValue`, you must migrate your React Native app to the New Architecture.
+
+## React 18 and the React Native New Architecture
+
+React 18 introduces [several new features](https://reactjs.org/blog/2022/03/29/react-v18.html) including:
+
+- Automatic batching
+- New Strict Mode behaviors
+- New hooks (`useId`, `useSyncExternalStore`)
+
+It also includes new concurrent features:
+
+- `startTransition`
+- `useTransition`
+- `useDeferredValue`
+- Full Suspense support
+
+The concurrent features in React 18 are built on top of the new concurrent rendering engine. Concurrent rendering is a new behind-the-scenes mechanism that enables React to prepare multiple versions of your UI at the same time.
+
+Previous versions of React Native built on the old architecture **cannot** support concurrent rendering or concurrent features. This is because the old architecture relied on mutating the native trees, which doesn’t allow for React to prepare multiple versions of the trees at the same time.
```

Comment:
```suggestion
Previous versions of React Native built on the old architecture **cannot** support concurrent rendering or concurrent features. This is because the old architecture relied on mutating the native trees, which doesn’t allow for React to prepare multiple versions of the tree at the same time.
```